federation-2/router-bridge/js-src/plan_worker.ts

```diff
@@ -191,10 +191,17 @@ async function run() {
         }
       } catch (e) {
         logger.warn(`an error happened in the worker runtime ${e}\n`);
+        let unexpectedError = {
+          name: "unexpectedError",
+          message: `${e}`,
+          extensions: {
+            code: "an error happened in the worker runtime",
+          },
```

Comment:
Building on the other quasi-standard properties of the GraphQL error ([see the specification](https://spec.graphql.org/draft/#sec-Errors.Error-result-format) here) maybe we want something like this:

```json5
"extensions": {
  "code": "QUERY_PLANNING_FAILED",
  "exception": {
    "stacktrace": [ /* I think this can be ...(err.toString().split(/\n)) ? */ ]
  }  
}
```

Reply:
I tried doing this. No idea if the syntax is correct.